<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerifyCertificateDialog</class>
 <widget class="QDialog" name="VerifyCertificateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Certificate Verification</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_warning">
     <property name="text">
      <string>&lt;b&gt;Warning: Certificate verification failed&lt;/b&gt;</string>
     </property>
     <property name="styleSheet">
      <string>color: red; font-size: 14px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_description">
     <property name="text">
      <string>The server certificate could not be verified. Please review the certificate details below and decide whether to accept this certificate.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Certificate Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_host_title">
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_host">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_port_title">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_port">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_common_name_title">
        <property name="text">
         <string>Common Name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_common_name">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_subject_title">
        <property name="text">
         <string>Subject:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_subject">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_issuer_title">
        <property name="text">
         <string>Issuer:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_issuer">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_fingerprint_title">
        <property name="text">
         <string>Fingerprint:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_fingerprint">
        <property name="text">
         <string>-</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
        <property name="font">
         <font>
          <family>Courier New</family>
         </font>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_question">
     <property name="text">
      <string>Do you want to accept this certificate?</string>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_reject">
       <property name="text">
        <string>&amp;Reject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_accept_temp">
       <property name="text">
        <string>Accept &amp;Temporarily</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_accept_perm">
       <property name="text">
        <string>Accept &amp;Permanently</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
